src/main/kotlin/no/nav/tms/min/side/proxy/dittnav/dittnavApi.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.tms.min.side.proxy.dittnav
+
+import io.ktor.application.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import io.ktor.util.pipeline.*
+import no.nav.tms.token.support.idporten.sidecar.user.IdportenUser
+import no.nav.tms.token.support.idporten.sidecar.user.IdportenUserFactory
+import org.slf4j.LoggerFactory
+
+fun Route.dittnavApi(consumer: DittnavConsumer) {
+
+    val log = LoggerFactory.getLogger(DittnavConsumer::class.java)
+
+    get("/dittnav/{proxyPath}") {
```

Comment:
dittnav-api bruker POST for å sende done-eventer, så det må også håndteres. Logikken blir vel identisk som for GET her. Må også legges til post i httpClient.

Reply:
Litt apropos dittnav-api er det nok en (liten?) stund før vi kan bruke proxy for denne. Tjenestene dittnav-api kaller støtter for det meste bare loginservice-token, og en kan ikke "nedgradere" fra loginservice til tokenx. Forhåpentligvis er dette klart før proxy går live.

Reply:
Hmmm, hvis det ikke er klart så kan vi kanskje gjøre to redirects fra frontenden slik at vi fortsatt støtter loginservice-tokens og la noe kall gå direkte til dittnav-api